### ISSUE TITLE
Make a better attempt at finding intended member for <seealso>

### ISSUE THREAD

**Issue by davedawkins** [fsprojects/FSharp.Formatting#789]
In this example
```fs
/// <summary>
/// Dispose all given items when the parent <c>SutilElement</c> is unmounted. Each item should implement <c>System.IDisposable</c>.
///
/// See also: <seealso cref="M:Sutil.DOM.unsubscribeOnUnmount"/>
/// </summary>
```

I'm not providing the full signature for unsubscribeOnUmount, because:
- it's extremely noisy (and ugly) - see below
- I'd have to look in a generated XML file

Actual signature
```
M:Sutil.DOM.unsubscribeOnUnmount(Microsoft.FSharp.Collections.FSharpList{Microsoft.FSharp.Core.FSharpFunc{Microsoft.FSharp.Core.Unit,Microsoft.FSharp.Core.Unit}})
```
It's all noise and friction.

Currently fsdocs makes an effort to find something to link to, but stops at the containing enitty (in this case `Sutil.DOM`). 

I've written a small amount of code that tries a little bit harder and goes on to find the actual member being specified. It stops at the first match, so overloads are an issue. Still, I think this adds value to the experience of writing docs.

I will send a PR.

**Comment by baronfel** [comment]
Another approach here would be to make the F# compiler always embed the fully qualified form, but let users write the simpler forms in the XMLDoc. This is what C# does and it feels very nice. Devs only have to worry about using the 'normal' names of things, and the compiler generates and uses the doc signature strings behind the scenes.

**Comment by davedawkins** [comment]
That would also be welcome. I've made the PR now.

**Comment by davedawkins** [comment]
This PR now included in 17.2.2
